### ISSUE TITLE
Alpha release is stuck at 1019 (from 14 June), does not autopublish anything after that

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#4487]
Seems like all of the new commits to master after 14 June are not going to alpha. The current alpha release is version number
1019 that was uploaded on 14 Jun 2021 22:12. Pinging @nicolas-raoul and @domdomegg for thoughts on what might have gone wrong with our autopublishing?

**Comment by nicolas-raoul** [comment]
Just a very random idea but maybe it is linked to CI failing recently?

**Comment by misaochan** [comment]
Ah, yes, it seems the last time CI ran on master was Commit e449240 16 days ago. We need to look into this. :/

**Comment by domdomegg** [comment]
Related to #4091

I think we need to get the secrets into the repository, then we should be able to recreate our Travis pipelines in GitHub actions fairly simply I think.

**Comment by sivaraam** [comment]
Just to be sure, in order to get the alpha builds back up is the only thing left is to contact WMF and set-up the credentials for the Actions workflow?

**Comment by misaochan** [comment]
Yes, I'm not familiar with how to recreate the Travis pipelines with Actions though. But if anyone would be willing to take this up, that would be great. :)

Once we have the PR up for the new workflows, I can contact WMF for the credentials, that was the final step when we did it previously.
